apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/StorageClientImpl.PatchObject.cs

```diff
@@ -42,6 +42,8 @@ private ObjectsResource.PatchRequest CreatePatchObjectRequest(Object obj, PatchO
             GaxPreconditions.CheckArgument(obj.Name != null, nameof(obj), "The Name property of the object to update is null");
             var request = Service.Objects.Patch(obj, obj.Bucket, obj.Name);
             options?.ModifyRequest(request);
+            RetryOptions retryOptions = options?.RetryOptions ?? RetryOptions.MaybeIdempotent(options?.IfMetagenerationMatch);
+            RetryHandler.MarkAsRetriable(request, retryOptions);
```

Comment:
Did we test this scenario as we have an `ApplyEncryptionKey` call after this? Is it the right place to inject the code?

Reply:
The "MarkAsRetriable" statement was previously inside the "ModifyRequest" and just moving it below and all tests are running fine. Also the ApplyEncryptionKey code doesnt seem to be affected by order .

Reply:
Ok. Just wanted you to be sure as I haven't looked at that code and so don't know.